app/src/main/java/org/oppia/android/app/mydownloads/downloads/DownloadsFragment.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.android.app.mydownloads.downloads
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import org.oppia.android.app.mydownloads.INTERNAL_PROFILE_ID_SAVED_KEY
+import javax.inject.Inject
+
+private const val CURRENT_SORT_TYPE_INDEX_SAVED_KEY =
+  "DownloadsFragment.current_sort_type_index"
+
+/** Fragment that contains downloaded topic list. */
+class DownloadsFragment :
+  InjectableFragment(),
+  SortByListIndexListener {
+
+  @Inject
+  lateinit var downloadsFragmentPresenter: DownloadsFragmentPresenter
+
+  private var previousSortTypeIndex: Int? = null
```

Comment:
Make it `private var previousSortTypeIndex: Int = -1` so that you can remove all null checks.

Reply:
done